activerecord/lib/active_record/connection_adapters/abstract/schema_definitions.rb

```diff
@@ -214,6 +214,7 @@ def initialize(types, name, temporary, options, as = nil)
         @name = name
       end
 
+      # Returns an array of ColumnDefinitions for the columns of the table.
```

Comment:
Yeah, I agree it's weird because `ColumnDefinition` is not documented, however many other places in this file comment about `ColumnDefinition` (including the next method, and `TableDefinition` class), and they are available in our docs. So I'm thinking it might be ok? Maybe we should doc `ColumnDefinition` instead - or review or docs to remove their mentions?

Nitpicking, but it's `ColumnDefinition`, without the `s`.